deployment/remove-dev-resources.ps1

```diff
@@ -53,9 +52,9 @@ az ad sp delete --id $aksServicePrincipal.appId
 # $l3ResourceGroup = $ApplicationName + "L3"
 # $l2ResourceGroup = $ApplicationName + "L2"
 
-# # Remove App resource group

```

Comment:
Should we update this to have "resource group" as "Resource Group", give it's a service name.